src/components/Categories.js

```diff
@@ -1,8 +1,21 @@
-const Categories = () => (
-  <section className="categories-section">
-    <h1 className="categories-progress">Coming Soon</h1>
-    <button type="button">Check Status</button>
-  </section>
-);
+import { useDispatch } from 'react-redux';
+import { statusCategories } from '../redux/categories/categories';
+
+const Categories = () => {
+  const dispatch = useDispatch();
+  return (
+    <section className="categories-section">
+      <h1 className="categories-progress">Coming Soon</h1>
+      <button
+        type="button"
+        onClick={() => {
+          dispatch(statusCategories());
+        }}
+      >
+        Check Status
+      </button>
+    </section>
+  );
+};
```

Comment:
- [ ] Nice job implementing `button for checking status`. However when we click on that button, the text `under construction` is not displayed. Kindly fix this issue and request another review.